lib/galaxy_test/driver/driver_util.py

```diff
@@ -563,7 +563,7 @@ def uvicorn_serve(app, port, host=None):
     import asyncio
     from uvicorn.server import Server
     from uvicorn.config import Config
-    config = Config(app, host=host, port=int(port))
+    config = Config(app, host=host, port=int(port), access_log=False)
```

Comment:
Nice! But for Galaxy tests the access logs are useful sometimes. Maybe that could be controlled with an environment variable ? `GALAXY_TEST_ACCESS_LOG` ? Then planemo can set this but we keep the access logs in the CI.

Reply:
sure. is there an example where an env var is used somewhere in the code base?

Reply:
https://github.com/mvdbeek/galaxy/blob/20d92afc5cae22d22528bbfd84eeadc94939efc5/lib/galaxy_test/driver/driver_util.py#L983